docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # -- Project information -----------------------------------------------------
 
 project = u'Dockstore'
-copyright = u'2019, OICR'
+copyright = u'2020, OICR'
 author = u'Andrew Duncan'
```

Comment:
We should update the author.

Reply:
To whom?

Reply:
Per Slack discussion, changed it to `OICR, UCSC`